src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -1,19 +1,17 @@
 import { Text } from '@konturio/ui-kit';
 import { i18n } from '~core/localization';
-import errorImage from './error-state-img.png';
 import s from './ErrorMessage.module.css';
 
 export function ErrorMessage({
   message,
-  marginTop = '30%',
+  margin = '32px 16px',
```

Comment:
Let's replace passing margin string with passing `className` or `containerClass`?
Default margin can go into the the s.spinner style (which probably needs to be renamed)